pkg/constant/constant_shared.go

```diff
@@ -82,8 +82,8 @@ const (
 	MetricsImageVersion                = "v0.6.1"
 	KubeProxyImage                     = "k8s.gcr.io/kube-proxy"
 	KubeProxyImageVersion              = "v1.25.0"
-	CoreDNSImage                       = "k8s.gcr.io/coredns/coredns"
-	CoreDNSImageVersion                = "v1.9.3"
+	CoreDNSImage                       = "docker.io/coredns/coredns"
```

Comment:
Don't know how long we want to wait for the image to appear in the k8s.io registry.